docs/source/guide/webhook_reference.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Webhook event format reference 
+short: Webhook Event Reference
+type: guide
+order: 653
+meta_title: Label Studio Webhook Event Reference 
+meta_description: Label Studio reference documentation for webhook event fields and payloads sent from Label Studio for integration with your machine learning pipeline. 
+---
+
+
+### Event reference
```

Comment:
This whole doc is still in progress, but I have what I need to create the event reference, which will be linked from the main webhooks doc table of available events.